tensorcircuit/translation.py

```diff
@@ -182,6 +182,16 @@ def _circuit_diagram_info_(self) -> List[str]:
             ci_name = gate_info["name"]
             cgate = CustomizedCirqGate(gatem, ci_name, len(index))
             cmd.append(cgate.on(*index))
+        elif gate_name == "measure":
+            cmd.append(cirq.MeasurementGate(key=gate_info["name"]).on(*index))
+        elif gate_name == "reset":
+            cmd.append(cirq.ResetChannel().on(*index))
+        elif gate_name in ["ox", "oy", "oz"]:
+            cmd.append(getattr(cirq, gate_name[1:])().controlled().on(*index))
```

Comment:
if no value specified, `controlled` is for cx cy cz gates?

Reply:
Hi! Theoretically that would be the case, as `Gate.controlled()` turns a gate (`x,y,z` in this case) into a controlled gate according to cirq's [documentation](https://quantumai.google/cirq/build/gates#other_gate_features).
Though I am still having some trouble trying to set up the development environment on my mac M2 machine, and haven't been able to test the gates yet.

Reply:
for m2, I guess this may help https://tensorcircuit.readthedocs.io/en/latest/contribs/development_Mac.html